Assets/Lib/Services/LoggingService.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace Lib.Services
+{
+    public class LoggingService
+    {
+        private string exceptionLogDir = Constants.LOG_FOLDER_EXCEPTIONS;
+        private string replayLogDir = Constants.LOG_FOLDER_REPLAY;
+        
+        public void LogException(string condition, string stacktrace)
+        {
+            string logMessage = String.Format("{0} -- {1}", condition, stacktrace);
+            using (StreamWriter sw = File.AppendText(exceptionLogDir+DateTime.Now.ToString("dd-MM-yyyy")+"_log.txt"))
+            {
+                sw.WriteLine($"{DateTime.Now.ToLongTimeString()} {DateTime.Now.ToShortDateString()} -- {logMessage}");
+            }
+        }
+
+        public Guid GetCarReplayGuid()
+        {
+            return Guid.NewGuid();
+        }
+
+        public void LogCarReplay(Guid guid, float timeInMs, Vector3 position, Quaternion rotation)
+        {
+            using (StreamWriter sw = File.AppendText(replayLogDir+guid+".txt"))
+            {
+                sw.WriteLine($"{timeInMs} -- {position} -- {rotation}");
+            }
```

Comment:
word hier de file open gehouden of geopend en weer meteen geclosed? misschien iets om te optimaliseren?